staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/admission.go

```diff
@@ -254,12 +300,12 @@ func (a *GenericAdmissionWebhook) callHook(ctx context.Context, h *v1alpha1.Exte
 			break
 		}
 	}
-	if !matches {
-		return nil
-	}
+	return matches
```

Comment:
`return false` here; `return true` instead of `break`; delete the `matches` var.

Reply:
Done.